chatbees/client_models/collection.py

```diff
@@ -205,14 +207,51 @@ def index_crawl(self, crawl_id: str):
         )
         Config.post(url=url, data=req.model_dump_json())
 
+    def configure_chat(
+        self,
+        persona: str = None,
+        negative_response: str = None,
+        temperature: float = None
```

Comment:
we did not hear any requirement for temperature. probably expose it only when required. not sure if other models will have the same ability. don't know if the customer really cares/wants to tune it.